docs/_chapters/920-faq.md

```diff
@@ -120,7 +120,7 @@ Inspired by the osgi-dev thread [Why default version for Export-Package is 0.0.0
 You can use either file. The reason there are two options is that one constituency thinks annotations are the only solution and the other is running pre Java 5 ... Basically if I would have to choose it would be packageinfo because that works anywhere but I expect that you probably would not like that :-) 
 
 ## Why are superclass not inspected for Component annotations? <a name="supercomps"/>
-Annotations are not inherited form the component's super classes. The problem is that bnd cannot in all cases can see super classes (and interfaces) because the super class does not have to be available in the build. In runtime you are sure the super class is there, at build time you do not have that luxury. So to make sure, always define your annotations on the actual component class.
+Annotations are not inherited form the component's super classes by default. The problem is that bnd cannot in all cases can see super classes (and interfaces) because the super class does not have to be available in the build. In runtime you are sure the super class is there, at build time you do not have that luxury. So to make sure, always define your annotations on the actual component class. Alternatively you can use the instruction `-dsannotations-options: inherit`. Then bnd use DS annotations found in the class hierarchy of the component class.
```

Comment:
> .. is that bnd cannot in all cases can see super classes ..

Can you remove the last `can`? ;) it was there before you modified but since you're here.

It should perhaps be noted that this is not true for the @Component annotation itself. This will not be inherited from super classes; causing a subclass to suddenly be a component.

Reply:
sure, done! Thx.